### PR TITLE
feat: add `double-navbar`

### LINKED ISSUE #67: Add `Navbars`
### Subject

Navbars

### Description

Add [Navbars](https://ui.mantine.dev/category/navbars/)

### Are you willing to participate in the development of this feature and create a pull request?

None

### PR DESCRIPTION
**Pull request by illionillion**
Related #67

### CODE REVIEW

**Review by quantumshiro — DISMISSED** [review]

**Review by fffk10 — APPROVED** [review]

File: contents/navigation/navbars/double-navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      <Flex flex={1}>
+        <VStack
+          alignItems="center"
+          as="aside"
+          flex="0 0 calc(3.75rem * 1)"
+          pt="md"
+        >
+          <Center>
+            <Avatar
+              src="https://avatars.githubusercontent.com/u/84060430?v=4"
```
Comment (taroj1205) [inline contents/navigation/navbars/double-navbar/index.tsx:71]: Please add `name="Hirotomo Yamada"`.

File: contents/navigation/navbars/double-navbar/metadata.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "en": {
+    "title": "DoubleNavbar",
```
Comment (taroj1205) [inline contents/navigation/navbars/double-navbar/metadata.json:3]: Make this `"Double Navbar"`.

**Review by hirotomoyamada — APPROVED** [review]

File: contents/navigation/navbars/double-navbar/metadata.json (hunk 2 above)
Comment (illionillion) [reply contents/navigation/navbars/double-navbar/metadata.json]: I fixed it.

File: contents/navigation/navbars/double-navbar/index.tsx (hunk 1 above)
Comment (illionillion) [reply contents/navigation/navbars/double-navbar/index.tsx:72]: I fixed it.

**Comment by illionillion** [comment]
I have fixed it as per your review.
@taroj1205

File: contents/navigation/navbars/double-navbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
+import {
+  BarChart3,
+  Calendar,
+  Fingerprint,
+  Gauge,
+  Home,
+  Settings,
+  User,
```
Comment (taroj1205) [inline contents/navigation/navbars/double-navbar/index.tsx:8]: I think `<BarChart3Icon />` might be preferable than `<BarChart3 />` and same for the others. What do you think?

Comment (illionillion) [reply contents/navigation/navbars/double-navbar/index.tsx:8]: OK, I will fix it.
Should the other icons be modified in the same way?

Comment (taroj1205) [reply contents/navigation/navbars/double-navbar/index.tsx:8]: Yes, please!

Comment (illionillion) [reply contents/navigation/navbars/double-navbar/index.tsx:8]: Thanks!
Let's create an issue because I think the same problem exists in other components.

Comment (taroj1205) [reply contents/navigation/navbars/double-navbar/index.tsx:8]: Sounds good, thanks!

Comment (illionillion) [reply contents/navigation/navbars/double-navbar/index.tsx]: I fixed it!

File: contents/navigation/navbars/double-navbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
+import {
+  BarChart3Icon,
+  CalendarIcon,
+  FingerprintIcon,
+  GaugeIcon,
+  HomeIcon,
+  Settings,
```
Comment (taroj1205) [inline contents/navigation/navbars/double-navbar/index.tsx:7]: Settings should also be `<SettingsIcon />`

File: contents/navigation/navbars/double-navbar/index.tsx (hunk 3 above)
Comment (illionillion) [reply contents/navigation/navbars/double-navbar/index.tsx]: I created issue.

https://github.com/yamada-ui/yamada-components/issues/280

File: contents/navigation/navbars/double-navbar/index.tsx (hunk 4 above)
Comment (illionillion) [reply contents/navigation/navbars/double-navbar/index.tsx]: Thanks.
I forgot. Fixed.

**Review by taroj1205 — APPROVED** [review]
Thanks!

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**